### PR TITLE
Add missing translations to actions, triggers and conditions of device automations

### PR DESCRIPTION
**Pull request by bieniu**
## Breaking change

None

## Proposed change

This PR adds the missing translations to actions, triggers and conditions of device automations

automations

## Type of change

- [ ] Dependency upgrade
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/config/automation/action/types/ha-automation-action-device_id.ts [hunk 1]
```diff
@@ -51,14 +51,18 @@ export class HaDeviceAction extends LitElement {
...
+        )}
       ></ha-device-picker>
       <ha-device-action-picker
         .value=${this.action}
         .deviceId=${deviceId}
         @value-changed=${this._deviceActionPicked}
         .hass=${this.hass}
-        label="Action"
+        label=${this.hass.localize(
+          "ui.panel.config.automation.editor.actions.name"
```
Comment (bramkragten) [inline src/panels/config/automation/action/types/ha-automation-action-device_id.ts:64]: This is plural, create a new key for this under the `device_id` key.

File: src/panels/config/automation/condition/types/ha-automation-condition-device.ts [hunk 2]
```diff
@@ -45,14 +45,18 @@ export class HaDeviceCondition extends LitElement {
...
+        )}
       ></ha-device-picker>
       <ha-device-condition-picker
         .value=${this.condition}
         .deviceId=${deviceId}
         @value-changed=${this._deviceConditionPicked}
         .hass=${this.hass}
-        label="Condition"
+        label=${this.hass.localize(
+          "ui.panel.config.automation.editor.conditions.name"
```
Comment (bramkragten) [inline src/panels/config/automation/condition/types/ha-automation-condition-device.ts:58]: Same

File: src/panels/config/automation/trigger/types/ha-automation-trigger-device.ts [hunk 3]
```diff
@@ -45,14 +45,18 @@ export class HaDeviceTrigger extends LitElement {
...
+        )}
       ></ha-device-picker>
       <ha-device-trigger-picker
         .value=${this.trigger}
         .deviceId=${deviceId}
         @value-changed=${this._deviceTriggerPicked}
         .hass=${this.hass}
-        label="Trigger"
+        label=${this.hass.localize(
+          "ui.panel.config.automation.editor.triggers.name"
```
Comment (bramkragten) [inline src/panels/config/automation/trigger/types/ha-automation-trigger-device.ts:58]: Same

File: src/panels/config/automation/action/types/ha-automation-action-device_id.ts (hunk 1 above)
Comment (bieniu) [reply src/panels/config/automation/action/types/ha-automation-action-device_id.ts]: Done.

**Review by bramkragten — APPROVED** [review]

**Pull request merged**